src/styles/blocks/main-block.scss

```
@@ -0,0 +1,47 @@
+* {
+  box-sizing: border-box;
+}
+
+.content {
+  @extend %align-ver-hor;
+  @extend %text-number;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0 $padding-sides;
+  gap: $gap-between-cell;
+  width: $widthCalendar;
+  @for $index from 1 through $all-days {
```

```suggestion
  width: $widthCalendar;
  
  @for $index from 1 through $all-days {
```
All is fine, I think you can make an empty line between

---

I agree

---

So do it :)